site/views/examples/forms-select-list.njk

```diff
@@ -3,11 +3,11 @@ layout: content
 title: Forms - Select List
 seoTitle: Custom select list examples based on ESL web components
 name: Select List
-tags: [examples, beta]
+tags: [examples, playground]
 icon: examples/select-list.svg
 aside:
-  components:
-    - esl-select-list
+components:
+- esl-select-list
```

Comment:
Incorrect, due to front etter is yml syntax the offsets are important
Before (in JSON):
```js
   aside: {
      components: ['esl-select-list']
   }
```
Now (in JSON):
```js
aside: ''
components: ['esl-select-list']
```
Or to be precise incorrect even by syntax (mix list/map declaration)